redbeams/src/main/resources/schema/app/20190508131601_CB-1174_database_server_config_env.sql

```diff
@@ -0,0 +1,20 @@
+-- CB-1174 database server config environment IDs
```

Comment:
We should store a CRN for every databaseServer as well. That will be the id to return.

Reply:
I will punt on this one for now, because I don't know yet how redbeams will determine the CRN for databases and database servers.